dev-aws/kafka-shared-msk/account-identity/account.tf

```
@@ -0,0 +1,109 @@
+import {
+  to = kafka_topic.account_identity_account_events_v2
+  id = "account-identity.account.events.v2"
+}
+
+import {
+  to = kafka_topic.account_identity_account_atomic_v1
+  id = "account-identity.account.atomic.v1"
+}
+
+import {
+  to = kafka_topic.account_identity_account_unified_events
+  id = "account-identity.account.unified.events"
+}
+
+import {
+  to = kafka_topic.account_identity_dev_account_events_anonymized_v0
+  id = "account-identity.dev.account.events.anonymized.v0"
+}
+
+import {
+  to = kafka_topic.account_identity_from_prod_account_events_anonymized_v0
+  id = "account-identity.from-prod.account.events.anonymized.v0"
+}
+
+import {
+  to = kafka_topic.account_identity_public_account_events
+  id = "account-identity.public.account.events"
+}
+
+import {
+  to = kafka_topic.account_identity_account_management_events
+  id = "account-identity.account.management.events"
+}
+
+resource "kafka_topic" "account_identity_account_events_v2" {
+  config = {
+    "compression.type" = "zstd"
+    "retention.ms"     = "604800000"
```

Please put a human readable comment for retention & also use tiered storage for more than 3 days, as stated in the [guide](https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/prod-aws/kafka-shared-msk/README.md#contributing): if you need to keep the data longer than 3 days, consider using [tiered storage](https://docs.aws.amazon.com/msk/latest/developerguide/msk-tiered-storage.html), and you need to decide how much of your data needs to be on the local hot storage. [Here](https://github.com/utilitywarehouse/kafka-cluster-config/blob/07f9b2be2c02aef072fe3900c3d4ee86fb0e282c/dev-aws/kafka-shared-msk/data-infra/data-infra.tf#L1-L16) is an example.
Suggestion:
```suggestion
    "remote.storage.enable" = "true"
    # keep data for 7 days
    "retention.ms"     = "604800000"
    # keep data in hot storage for 1 day
    "local.retention.ms" = "86400000"    
```